build/gulp/tasks/docs.ts

```diff
@@ -53,7 +49,6 @@ task('clean:docs:example-menus', cb => {
 task(
   'clean:docs',
   parallel(
-    'clean:docs:component-info',
```

Comment:
We shouldn't clear directory for caching purposes

Reply:
So there is no way how to clear the cache (other than deleting the files manually)? :-(

Reply:
Yep. We can make a separate script (in `package.json` for this, delete DLL, doc definitions, clear dist), something like `yarn cleanup`. What do you think?

Reply:
...and we can do that in a separate PR